src/api/api.js

```diff
@@ -43,6 +43,10 @@ class Api {
         errors.push(`${name} parameter is not defined`);
       } else {
         req.body[name] = parseInt(req.body[name], 10);
```

Comment:
Outside the scope of this PR, but do you have to handle the case where the string is not parseable and `parseInt()` returns NaN?

Reply:
Good point, will fix this shortly